R/grid.R

```diff
@@ -65,13 +65,29 @@ draw_custom_grid <- function(object, view.name){
   grid.args <- remove_field(grid.args, "equilogs")
   at <- list()
   for (side.name in side.names){
-    usr.at <- axis_axTicks(object, as.side(side.name))
-    if (is.null(usr.at))
-      at[[side.name]] <- grid_axTicks(object, as.side(side.name))
-    else
-      at[[side.name]] <- usr.at
+    side.axis <- as.axis(side.name)
+    if(side.axis == "x") 
```

Comment:
I would love it if we could use curly brackets for all if else statements....I know we don't have an official required style...but it's so much easier to follow in my opinion.